cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -85,7 +89,11 @@ spec:
               mountPath: /etc/nginx/conf.d/
           resources:
 {{ toYaml .Values.kubecostFrontend.resources | indent 12 }}
+        {{- if .Values.kubecostFrontend.imagePullPolicy }}
```

Comment:
Looks like this may be missing corresponding parameter in values.yaml?

Reply:
Thanks!

Reply:
It would still be ok since it would default to Always, but yeah, we can add it to values as well